ansible/roles/docker/templates/centos7.Dockerfile.j2

```diff
@@ -25,6 +25,8 @@ RUN yum -y update && \
       python3-pip \
       fontconfig-devel
 
+RUN yum -y install libtool automake
```

Comment:
Is there a particular reason (layer caching?) to keep this separate from the earlier package installs?

Reply:
Only that it was for a separate purpose from the main node build - happy to unify if preferred

Reply:
Adjusted :-)